Makefile

```diff
@@ -134,6 +134,23 @@ generate-bundle: regenerate $(OPM)
 	$(OPERATOR_SDK) bundle validate --verbose bundle
 .PHONY: generate-bundle
 
+# Generate bundle manifests and metadata, then validate generated files.
+VERSION ?= 4.7.0
+# Options for 'bundle-build'
+ifneq ($(origin CHANNELS), undefined)
+BUNDLE_CHANNELS := --channels=$(CHANNELS)
+endif
+ifneq ($(origin DEFAULT_CHANNEL), undefined)
+BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
+endif
+BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
```

Comment:
It looks like these conditionals are used only by the `bundle` target. Can we group them there?

We need to discuss here, how these `BUNDLE_CHANNELS` is populated in general.